compilation error due to double symbol 'test'

Getting this error:

``/usr/bin/ld: build/extern/src/ym2151.o:/home/irmen/Projects/parked/x16-emulator/src/extern/src/ym2151.c:86: multiple definition of `test'; build/main.o:/home/irmen/Projects/parked/x16-emulator/src/main.c:130: first defined here
collect2: error: ld returned 1 exit status``

from a quick search, it seems that the 'test' variable in main.c isn't even used anywhere
but that was with a text search maybe someone can confirm with a proper C/C++ IDE dependency search.

--- a/src/main.c
+++ b/src/main.c
@@ -127,8 +127,6 @@ bool prg_finished_loading;
 int prg_override_start = -1;
 bool run_after_load = false;
 
-bool test = true;
-
 char *nvram_path = NULL;
 
 #ifdef TRACE

